documentation-site/pages/components/table.mdx

```diff
@@ -0,0 +1,57 @@
+<!--
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+import Example from '../../components/example';
+import API from '../../components/api';
+import Layout from '../../components/layout';
+
+import Basic from 'examples/table/basic.js';
+import VerticalScroll from 'examples/table/vertical-scroll.js';
+import HorizontalScroll from 'examples/table/horizontal-scroll.js';
+import Virtual from 'examples/table/virtual.js';
+import VirtualHorizontal from 'examples/table/virtual-horizontal-scroll.js';
+
+export default Layout;
+
+# Table
+
+Display larger amounts of information in a scannable UI.
+
+## Examples
+
+<Example title="Basic Usage" path="examples/tag/basic.js">
+  <Basic />
+</Example>
+
+<Example title="Vertical Scroll" path="examples/table/vertical-scroll.js">
+  <VerticalScroll />
+</Example>
+
+<Example title="Horizontal Scroll" path="examples/table/horizontal-scroll.js">
+  <HorizontalScroll />
+</Example>
+
+<Example
```

Comment:
should we put a bit of explanation here when someone should use a virtualized table?